jobs/ostree-compose.yml

```diff
@@ -59,40 +59,30 @@
               which repo of the package being updated
     builders:
         - shell: |
-            cat << EOF > ${WORKSPACE}/job.properties_running
-                topic=org.centos.prod.ci.pipeline.compose.running
-                build_url=$BUILD_URL
-                compose_url=http://artifacts.ci.centos.org/artifacts/fedora-atomic/${fed_branch}/ostree
-                compose_rev=
-                build_id=$BUILD_ID
-                ref=fedora/${fed_branch}/x86_64/atomic-host
-                rev=${rev}
-                namespace=rpms
-                repo=$fed_repo
-                status=success
-                test_guidance=
-            EOF
+            if [ "$fed_branch" = "master" ]; then
+                echo "fed_branch=rawhide" > ${WORKSPACE}/job.properties
+            fi
         - inject:
-            properties-file: ${WORKSPACE}/job.properties_running
+            properties-file: ${WORKSPACE}/job.properties
         - jms-messaging:
             override-topic: org.centos.prod.ci.pipeline.compose
```

Comment:
So I think in the cases where you know topic (which here you do as per line 72), you can put the full topic here.  The reason why I abridged mine was because mine is variable (mainly because it could be .ignore .queued etc), and $topic is not supported in override-topic yet, so I had to leave off the end (that was my thinking at least, I could have left it cut off in unnecessary places in my code).